pioinstaller/penv.py

```
@@ -61,15 +92,86 @@ def get_uv_platform():
+
+        checksums = {}
+        for asset in release_data.get("assets", []):
+            asset_name = asset.get("name", "")
+            if asset_name.endswith((".tar.gz", ".zip")):
+                # GitHub now provides SHA256 digests in the API response
+                digest = asset.get("digest", {}).get("sha256")
+                if digest:
+                    checksums[asset_name] = digest
+
+        log.debug("Fetched checksums for %d assets", len(checksums))
+        return checksums
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)